doc/advanced-configuration/Services.md

```diff
@@ -1,80 +1,101 @@
 Some functionalities in MongooseIM are provided by "services".
-A service is similar to a module, but while a module is started for every virtual host and may have global or host-specific configuration, a service is started only once with global configuration.
-Service configuration is similar to a module configuration, e.g.:
-```
-{services, [
-            {service_admin_extra, [{submods, [node, accounts, sessions]}]},
-            {service_mongoose_system_metrics, [report,
-                                               {intial_report, 300000},
-                                               {periodic_report, 108000000}]}
-           ]
-}.
-```
+A service is similar to a module, but while a module is started for every virtual 
+host and may have global or host-specific configuration, a service is started 
+only once with global configuration.
 
+* **Scope:** global
+* **Syntax:** Options for each service are specified in a `[[services]]` array.
+* **Default:** None - each service needs to be enabled explicitly.
+Typical services are already specified in the example configuration file.
+* **Example:** A configuration of the `service_admin_extra` service.
 
-## Service list
+```
+[[services]]
+  name = "service_admin_extra"
+  submods = ["node", "account", "sessions", "vcard", "gdpr", "upload", "roster",
+             "last", "private", "stanza", "stats"]
+```
```

Comment:
I'm not sure if I'm missing something, but it's not the way we configure `services`. [Example](https://github.com/esl/MongooseIM/blob/toml-config/test/config_parser_SUITE_data/mongooseim-pgsql.toml#L208)
You can reference the [parser](https://github.com/esl/MongooseIM/blob/toml-config/src/config/mongoose_config_parser_toml.erl#L597) as well

Reply:
Thx, got confused (copied from...) by an old document on Confluence. Should be fixed now